main.c

```diff
@@ -134,6 +134,7 @@ int main() {
                 //Additional info
                 printf("Any additional info?\n");
                 gets(addInfo);
+                state++;
```

Comment:
M-am incurcat eu, era ok inainte. Sa faci un commit in care faci revert la schimbare si dupa cred ca iti pot face merge.